modules/bullet/rigid_body_bullet.cpp

```diff
@@ -240,7 +240,7 @@ void RigidBodyBullet::KinematicUtilities::copyAllOwnerShapes() {
 				shapes.write[i].shape = static_cast<btConvexShape *>(shape_wrapper->shape->create_bt_shape(owner_scale * shape_wrapper->scale, safe_margin));
 			} break;
 			default:
-				WARN_PRINT("This shape is not supported to be kinematic!");
+				WARN_PRINT("This shape is not supported to be kinematic.");
```

Comment:
That warning message doesn't make much sense to me. Shouldn't it be `This shape is not supposed to be kinematic.` or `Kinematic shapes are not supported.`?

Reply:
Either of those, yeah. Whoever wrote this was definitely not a native English speaker and mixed up two sentences.

Reply:
`This shape is not supported for kinematic collision.` sound good.